File: AED-2/Arvores/bst/TAD-flexivel/matriz-flexivel/java/inserir.java
package matriz.java;

public class inserir {
    void inserir(int l, int c, int valor) {
        Celula temp = inicio;
        for (int i = 0; i < l; i++) temp = temp.inf;
        for (int j = 0; j < c; j++) temp = temp.dir;
        temp.elemento = valor;
    }
}

File: AED-2/Arvores/bst/TAD-flexivel/matriz-flexivel/c/inserir.c
void inserirValor(Matriz* mat, int linha, int coluna, int valor) {
    Celula *temp = mat->inicio;
    for (int i = 0; i < linha; i++) temp = temp->inf;
    for (int j = 0; j < coluna; j++) temp = temp->dir;
    temp->elemento = valor;
}
